java/core/src/main/scala/io/pdal/PackedPoints.scala

```diff
@@ -0,0 +1,66 @@
+package io.pdal
+
+import java.nio.ByteBuffer
+import java.util
+
+import scala.collection.JavaConversions._
+
+/**
+  * PackedPoints abstraction to work with packed point(s) in JVM memory.
+  * SizedDimType contains size and offset for a particular packed point with the current set of dims.
+  **/
+case class PackedPoints(bytes: Array[Byte], dimTypes: util.Map[String, SizedDimType], metadata: String = "", schema: String = "") {
```

Comment:
@pomadchin  Does it make sense to call this class "PointCloud"? It seems like the main interface into how we work with PointClouds on the JVM, and the fact that they are backed by a byte array of "packed" points feels like an implementation detail that might confuse users (or give less of an eyebrow raise if it was called something that's more in the domain vernacular).